core/trino-main/src/main/java/io/trino/sql/analyzer/FeaturesConfig.java

```diff
@@ -1103,4 +1104,17 @@ public FeaturesConfig setDisableSetPropertiesSecurityCheckForCreateDdl(boolean d
         this.disableSetPropertiesSecurityCheckForCreateDdl = disableSetPropertiesSecurityCheckForCreateDdl;
         return this;
     }
+
+    public boolean isIncrementalHashArrayLoadFactorEnabled()
+    {
+        return incrementalHashArrayLoadFactorEnabled;
+    }
+
+    @Config("incremental-hash-array-load-factor.enabled")
```

Comment:
Can we have it deprecated and plan to remove it after some time?

Reply:
ok